source/features/navigation/store.ts

```diff
@@ -61,6 +61,20 @@ export class NavigationStore extends Store {
     );
   }
 
+  private updateGAEvents() {
+    try {
+      const windowObject: Window | any = window;
```

Comment:
`any` is basically the same as ignoring the line

Reply:
Fixed